test/board_test.rb

```diff
@@ -0,0 +1,73 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/board'
+require './lib/cell'
+require './lib/ship'
+require 'pry'
+
+class BoardTest < MiniTest::Test
+  def test_it_exists
+    board = Board.new
+    assert_instance_of Board, board
+  end
+
+  def test_it_has_readable_attributes
+    board = Board.new
```

Comment:
How do you feel about putting a line between our board declaration and asserts?

Reply:
Good catch! line removed.